Doc/library/stdtypes.rst

```diff
@@ -5314,8 +5314,9 @@ instantiated from the type::
 Other Built-in Types
 ====================
 
-The interpreter supports several other kinds of objects. Most of these support
-only one or two operations.
+The interpreter supports several other kinds of objects, including instances
+of the :class:`object` class itself. Most of these support only one or two
```

Comment:
This addition reads strangely, I can’t really make sense of it.

(The section is about other classes (other than the ones already described in the page), and the new fragment talks about instances of object? what are these? what does it mean that the intepreter supports them?)

I would revert this too.